examples/docker/vizro/Dockerfile

```diff
@@ -1,24 +1,14 @@
 FROM python:3.11-slim
 
-# Set environment variables
```

Comment:
As far as I could tell these weren't actually necessary for anything. e.g. `PIP_NO_CACHE_DIR=off` is overridden below anyway by `--no-cache-dir`.